lib/ansible/modules/cloud/amazon/cloudwatchlogs_log_group.py

```diff
@@ -223,10 +242,11 @@ def main():
         kms_key_id=dict(required=False, type='str'),
         tags=dict(required=False, type='dict'),
         retention=dict(required=False, type='int'),
+        purge_retention_policy=dict(required=False, type='bool'),
```

Comment:
I think we need this could stay with the ``default=False``, as far as I can tell mutually_exclusive is evaluated before the defaults are put into place.